uui/components/datePickers/RangeDatePickerBody.tsx

```diff
@@ -88,28 +88,29 @@ export class RangeDatePickerBody extends React.Component<RangeDatePickerBodyProp
         activePart: null,
     };
 
-    getDayCX = (day: Dayjs) => {
+    getDayCX = (day: Dayjs): string[] => {
         const dayValue = day.valueOf();
-        const fromValue = this.props.value?.selectedDate.from ? dayjs(this.props.value.selectedDate.from).valueOf() : null;
-        const toValue = this.props.value?.selectedDate.to ? dayjs(this.props.value.selectedDate.to).valueOf() : null;
+        const fromValue = this.props.value.selectedDate?.from ? dayjs(this.props.value.selectedDate.from).valueOf() : null;
+        const toValue = this.props.value.selectedDate?.to ? dayjs(this.props.value.selectedDate.to).valueOf() : null;
 
-        const inRange = dayValue >= fromValue && dayValue <= toValue && fromValue !== toValue && fromValue && toValue;
+        const inRange = fromValue && toValue && dayValue >= fromValue && dayValue <= toValue && fromValue !== toValue;
         const isFirst = dayValue === fromValue;
         const isLast = dayValue === toValue;
 
-        return [
+        return [cx(
```

Comment:
Are you sure that wee ned to have `cx` call here? I guess the `cx` will be called under the result of this function in place we `getDayCX` was called

Reply:
getDayCX should be passed as `(day: Dayjs): string[];`. but currently we have `(day: Dayjs) => (string | false | 0 | null)[]`. To meet the type I call `cx` here.